variable/include/scipp/variable/variable.tcc

```diff
@@ -785,7 +785,11 @@ Variable Variable::create(const units::Unit &u, const Dims &d, const Shape &s,
 
 /// Macro for instantiating classes and functions required for support a new
 /// dtype in Variable.
-#define INSTANTIATE_VARIABLE(...)                                              \
+#define INSTANTIATE_VARIABLE(name, ...)                                        \
```

Comment:
Why variadic? Shouldn't we only have a single second argument to denote a single type

Reply:
Templates such as `std::pair` that contain a comma are recognizedas two arguments by the preprocessor. We don't have any of those right now, but we did in the past.